server/pairing/payload_management.go

```diff
@@ -130,8 +130,8 @@ func (rmm *RawMessagePayloadMarshaller) MarshalProtobuf() (data []byte, err erro
 	syncRawMessage := new(protobuf.SyncRawMessage)
 
 	syncRawMessage.RawMessages = rmm.payload.rawMessages
-	if len(rmm.payload.subAccounts) > 0 {
-		syncRawMessage.SubAccountsJsonBytes, err = json.Marshal(rmm.payload.subAccounts)
+	if rmm.payload.profileKeypair != nil && len(rmm.payload.profileKeypair.KeyUID) > 0 {
+		syncRawMessage.SubAccountsJsonBytes, err = json.Marshal(rmm.payload.profileKeypair)
```

Comment:
rename SubAccountsJsonBytes to ProfileKeypairJsonBytes ?

Reply:
Answered in (this comment](https://github.com/status-im/status-go/pull/3475#discussion_r1198659377), prefer to handle that in a new PR, too much for this one cause that change will require many other changes, as well as client change.